.bazelrc

```diff
@@ -1,2 +1,5 @@
 # Include git version info
 build --workspace_status_command hack/print-workspace-status.sh
+
+# bazel including rules_docker 0.12.0 may not need the following flag
```

Comment:
We should also investigating bumping our dependencies to get around this in future 😀